Quality/MK8_1080p/rules.txt

```diff
@@ -18,6 +18,22 @@ formatsExcluded = 0x41A # exclude obvious textures
 overwriteWidth = 960
 overwriteHeight = 540
 
+# Multiplayer-Mode Resolution (2p)
+[TextureRedefine]
+width = 640
+height = 720
+formatsExcluded = 0x41A # exclude obvious textures
+overwriteWidth = 960
+overwriteHeight = 1080
+
+# Multiplayer-Mode Resolution (4p)
+[TextureRedefine]
+width = 640
+height = 360
+formatsExcluded = 0x41A # exclude obvious textures
+overwriteWidth = 960
+overwriteHeight = 1080
```

Comment:
Shouldn't this be lower than the width? Should be half.

ALSO: this conflicts with the half alpha res rule. Here are two possibilities

- Assume half-alpha rule will deal with 4p render target, but at great performance cost (at 4k, you'd be pushing 4x 4k targets)
- Make half res target rescale to half of the target resolution. Will make single player use half res alpha like the original game, but at least it won't be half of 720p.
- Different graphics packs for 4p multiplayer and single player.

Have these sizes been verified with a graphics debugger? I thought they might actually be uneven due to the black bars.